components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -298,6 +298,14 @@ To finish up, return to your Chef Habitat Builder on-prem installation and view
 
 Chef Habitat Builder uses the same mechanisms that Chef Automate does for [backups]({{< relref "backup.md" >}}), [log management]({{< relref "log-management.md" >}}), and [uninstalling]({{< relref "troubleshooting.md#uninstalling-chef-automate" >}}).
 
+### Chef Habitat Environment Variables
+
+Set the Chef Habitat Builder on-prem `HAB_BLDR_URL` environment variable on the command line with:
```

Comment:
Might want to mention when someone would want to do this.

Reply:
```suggestion
If needed for troubleshooting or communicating with your Chef Habitat Builder on-prem installation, set the `HAB_BLDR_URL` environment variable on the command line with:
```